docs/install_guides/installing-openvino-conda.md

```diff
@@ -45,18 +47,11 @@ This guide provides installation steps for Intel® Distribution of OpenVINO™ t
    ```
 4. Verify the package installed:
    ```sh
-   python -c "import openvino"
+   python -c "from openvino.inference_engine import IECore"
```

Comment:
Executing this line will cause error "No module named 'numpy'" (INST-19812). Suggestion: add info about how to deal with it, e.g. in case of error "No module named 'numpy'" execute "conda install numpy"

Reply:
Add install prerequisites step for this as a temporary solution for 2021.4 release documentation update.
This should be remove for next release, as it shall be implemented as package explicit dependency.